cmd/paymentmethods/list_entity.go

```diff
@@ -63,6 +63,6 @@ var listEntityPaymentMethodsCmd = &cobra.Command{
 }
 
 func init() {
-	rootCmd.AddCommand(listEntityPaymentMethodsCmd)
-	listEntityPaymentMethodsCmd.Flags().StringP(utils.EntityIdFlag, "", "", "Entity ID. Uses environment variable if blank")
+	Cmd.AddCommand(listEntityPaymentMethodsCmd)
```

Comment:
This print out includes the request object and a separate pagination_params
`{"payment_methods":[],"request":{"entity_id":"ba65079e-...-89041fa232b1","pagination_params":null}}`

Reply:
Removed all request object info